static/js/components/LearnerSearch.js

```diff
@@ -142,6 +173,7 @@ export default class LearnerSearch extends SearchkitComponent {
                     role="button"
                     id="email-selected"
                     className="mm-button"
+                    onClick={this.openEmailComposer}
```

Comment:
one way to get rid of the `openEmailComposer()` definition above is to add `openEmailComposer` to the list of `const` variables and rewrite this line as `_.bind(openEmailComposer, this, this.searchkit)` or `_.partial(openEmailComposer, this.searchkit`. ramda might have some equivalent as well